content/5_Plat/Bin_Jump.mdx

```diff
@@ -353,6 +353,312 @@ int main() {
 
 <IncompleteSection />
 
+### Solution (Company II)
+
+First, let $\texttt{tI}, \texttt{tO}, and \texttt{up}$ be the time-in, time-out, and ancestor table for all nodes in the tree. 
```

Comment:
To be more consistent with euler tour module you should probably use st en for timer.